src/checkers/inference/model/serialization/ToStringSerializer.java

```diff
@@ -243,6 +265,11 @@ public String serialize(CombVariableSlot slot) {
         return sb.toString();
     }
 
+    @Override
+    public String serialize(ArithmeticVariableSlot slot) {
+        return serialize((VariableSlot) slot);
```

Comment:
Instead of directly using `serialize(VariaableSlot)` method, should we at least marks this `slot` is an `ArithmeticVarSlot`? Otherwise how can users distinguish `VarSlot` and `ArithmeticVarSlot` from the string serialization in this class?

Reply:
It is unnecessary as the debug solver always prints out a line saying what kind of slots it is followed by all slots that belong in that category:

```
Created VariableSlot
10: merged to -> [CombVariableSlot(13)]
:AstPath:
AstPathLocation( Variables.null.a:Variables::a:Variable.type )

6
:AstPath:
MissingLocation

7
:AstPath:
ClassDeclLocation( Variables )

Created RefinementVariableSlot

Created ExistentialVariableSlot
8: ( 7 | 5 ) 
:AstPath:
MissingLocation

Created CombVariableSlot
13: merges [VariableSlot(10), VariableSlot(9)]
14: merges [VariableSlot(12), VariableSlot(11)]

Created ConstantSlot
@PolyAll(1)
@PolyUnit(2)
@UnitsInternal(3)

Created ArithmeticVariableSlot
21
:AstPath:
AstPathLocation( Variables.Main()V.null:Variables:Main()V::Method.body, Block.statement 1, Variable.initializer )

```

Reply:
Discussed personally with Jeff. Maybe remove this method.